contents/handbook/engineering/on-call-rotation.md

```diff
@@ -0,0 +1,49 @@
+---
+title: On-call rotation
+sidebar: Handbook
+showTitle: true
+---
+
+Every week, three engineers are on call in shifts. If you're one of the lucky three, congratulations!
+
+PostHog Cloud doesn't shut down at night (_whose_ night anyway?) nor on Sunday. As a 24/7 service, our goal is to be 100% operational 100% of the time. The universe doesn't always agree and hiccups do occur, which we catch in the form of service alerts. To make sure no alert goes unhandled, there is **always** one engineer on call, tasked with first-line defense.
+
+## When is my turn?
+
+We're very lucky that we have an almost-follow-the-sun rotation for service alerts, so nobody gets woken up in the middle of the night. Every engineer at PostHog is expected to be part of the rotation within one of the three PagerDuty schedules:
+
+- [Europe](https://posthog.pagerduty.com/schedules#PF7ZGBT) (08:00 to 16:00 UTC)
+- [Americas East](https://posthog.pagerduty.com/schedules#PW1E9Y4) (16:00 to 00:00 UTC)
+- [Americas West](https://posthog.pagerduty.com/schedules#P3J10CZ) (00:00 to 08:00 UTC)
+
+Before going on call, make sure you have the [PagerDuty mobile app](https://support.pagerduty.com/docs/mobile-app) installed and configured. This way it'll be harder to miss an alert.
+
+## What if I'm scheduled for a week when I won't be available?
+
+Swap with another engineer in advance! Find a volunteer by asking in Slack, then use PagerDuty schedule overrides. You can trade whole weeks, but also just specific days. Remember not to alter the rotation's core order, as that's an easy way to accidentally shift the schedule for everyone.
+
+## What do I do on call?
+
+You just go about your day, paying no attention to being on call most of the time! But keep your laptop at hand, because when you do get paged, it's time to switch gears immediately and start investigating the alert.
+
+To be ready, make sure you have access to:
```

Comment:
My first stop when I get a page (or I see something on Slack) is almost always Grafana, so my suggestion would be to add links to http://grafana-prod-us/dashboards and http://grafana-prod-eu/dashboards like you have for the admin interfaces and Metabase.  Thanks so much for doing this!  It looks great!

Reply:
Added it below 👍